homeassistant/components/huawei_lte/strings.json

```diff
@@ -1,20 +1,19 @@
 {
   "config": {
     "abort": {
-      "already_configured": "This device has already been configured",
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]",
       "already_in_progress": "This device is already being configured",
       "not_huawei_lte": "Not a Huawei LTE device"
     },
     "error": {
-      "connection_failed": "Connection failed",
```

Comment:
Was this error not used in the config flow?

Reply:
Nope. Couldn't find it in config flow or tests.